react/features/token-data/actions.js

```diff
@@ -29,8 +29,14 @@ export function parseTokenData() {
         // the iss field is not anonymous. If you want to pass data with
         // JWT token but you don't want to pass the JWT token for
         // verification the iss field should be set to "anonymous".
-        if (!config.token && payload && payload.iss !== 'anonymous') {
-            config.token = token;
+        if (payload && payload.iss !== 'anonymous') {
+            parsedToken.token = token;
```

Comment:
I looked through to code and we're not using jwt property of token data. So we can get rid of it and store only parsed/non-parsed token and `token` property only if issuer is not anonymous or corresponding property was sent in config.